verification/cbmc/proofs/aws_string_eq_ignore_case/aws_string_eq_ignore_case_harness.c

```diff
@@ -9,9 +9,10 @@
 #include <proof_helpers/utils.h>
 
 void aws_string_eq_ignore_case_harness() {
-    struct aws_string *str_a = nondet_bool() ? ensure_string_is_allocated_bounded_length(MAX_STRING_LEN) : NULL;
-    struct aws_string *str_b =
-        nondet_bool() ? (nondet_bool() ? str_a : NULL) : ensure_string_is_allocated_bounded_length(MAX_STRING_LEN);
+    struct aws_string *str_a = ensure_string_is_allocated_bounded_length(MAX_STRING_LEN);
```

Comment:
`ensure` is not the right name anymore, given that it can return `null`

Reply:
I replaced all instances with `nondet_allocate_string_bounded_length`.